app/services/create_budget.rb

```diff
@@ -24,11 +25,16 @@ def call(attributes: {})
     result
   end
 
-  private def editable_report_for_activity(activity:)
+  private
+
+  def editable_report_for_activity(activity:)
     Report.find_by(organisation: activity.organisation, fund: activity.associated_fund, state: Report::EDITABLE_STATES)
   end
 
-  private def sanitize_monetary_string(value:)
-    Monetize.parse(value)
+  def convert_and_assign_value(budget, value)
+    budget.value = ConvertFinancialValue.new.convert(value.to_s)
+    budget
```

Comment:
What's the reason for returning the record in each of these methods? The result of `convert_and_assign_value` does not appear to be used.

Reply:
Good point, it's updated in place isn't it. I've removed them.

Reply:
Cool cool :shipit: